docs/api.rst

```diff
@@ -4644,6 +4648,7 @@ Dealing with History Events
                       "notes": "Deposit 32 ETH to validator 4242",
                       "sequence_index": 15,
                       "timestamp": 1642802807,
+                      "has_accounting_rule": false,
```

Comment:
Wouldn't it be simpler and easier for both frontend and api if this is only existing if rule is missing and be `"missing_accounting_rule": true,`